eng/common/templates/job/execute-sdl.yml

```diff
@@ -29,6 +29,12 @@ jobs:
   pool:
     name: Hosted VS2017
   steps:
+  - task: UseDotNet@2
+    displayName: Use .NET Core Sdk (2.2.0)
+    inputs:
+      packageType: sdk
```

Comment:
I believe you only need the runtime to be installed, so perhaps `packageType` should be `runtime`?